superset-frontend/src/explore/components/ExploreViewContainer/index.jsx

```diff
@@ -175,6 +175,19 @@ const updateHistory = debounce(
       additionalParam[URL_PARAMS.datasetId.name] = datasetId;
     }
 
+    const urlParams = payload?.url_params || {};
```

Comment:
Wouldn't be cleaner to just initialize `additionalParam` with `payload?.url_params`? If some of the parameters are already in the URL they will be overridden anyway. This way you don't need the `forEach` and the `if` check.

Reply:
> Wouldn't be cleaner to just initialize additionalParam with payload?.url_params?

1. `formDataKey` in urlParams
2.  it isn't known which url_params don't need to be put into `additionalParams`


> If some of the parameters are already in the URL they will be overridden anyway. This way you don't need the forEach and the if check.

﻿﻿There needs to be a pattern that controls which params can be put in, and which do not.